blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -51,9 +51,9 @@ const HeaderAccountAction = ({ customFields }) => {
     return () => { isActive = false; return null; };
   }, [Identity, loggedIn]);
 
-  const handleLogout = () => {
-    Identity.logout().then(() => { setIsLoggedIn(false); setUser(null); });
-  };
+  // const handleLogout = () => {
```

Comment:
Should this be commented out?

Reply:
I'm not sure. is this within scope of the ticket?

Reply:
I would assume so - the ticket is for the logged out state of the block - I would take it that it would all be working as expected. I don't see another ticket that's to follow that would add this logic